projects/packages/sync/src/class-queue.php

```diff
@@ -375,9 +383,9 @@ public function checkout_with_memory_limit( $max_memory, $max_buffer_size = 500
 			// @codingStandardsIgnoreEnd
 		}
 
-		if ( count( $items ) === 0 ) {
+		$items_count = is_countable( $items ) ? count( $items ) : 0;
```

Comment:
Perhaps you should do something to stop line 380 from raising a warning if `$items` is not an array too?